cfme/tests/ansible/test_embedded_ansible_automate.py

```diff
@@ -419,3 +421,43 @@ def test_variable_pass(request, appliance, setup_ansible_repository, import_data
         provision_request = appliance.collections.requests.instantiate(request_description)
         provision_request.wait_for_request(method="ui")
         request.addfinalizer(provision_request.remove_request)
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1677575])
+@pytest.mark.parametrize(
+    "import_data",
+    [DatastoreImport("bz_1677575.zip", "bz_1677575", None)],
+    ids=["datastore"],
+)
+def test_import_export_domain_with_ansible_method(request, appliance, setup_ansible_repository,
+                                                  import_data):
+    """This test case tests support of Export/Import of Domain with Ansible Method
+
+    Bugzilla:
+        1677575
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseimportance: high
```

Comment:
**Required:**
@ganeshhubale No need to mention these Polarion tags which are by default. --> `caseimportance: high`, `caseposneg: positive` and `testtype: functional`

Reply:
Added it for more clarity while looking into doc string. Although we have choice to keep it in polarion doc or remove.

Reply:
@ganeshhubale Afaik, this isn't doc string, these are parameters for Polarion to set. We kept few parameters by default to make clarity.